### PR TITLE
docs: Mention #argo-wf-sustainability in sustainability efforts

### PR DESCRIPTION
**Pull request by terrytangyuan**
This is a new dedicated channel for this effort.

### CODE REVIEW

**Comment by juliev0** [comment]
I was thinking about this as well. If I leave it as a Private channel, how do you think they should make the request? Through the `argo-wf-contributors` slack channel? Or maybe I should make it a Public channel and have them join themselves?

**Comment by terrytangyuan** [comment]
It should remain as private channel. They can submit a PR or ask us in wf contributors channel.

**Comment by juliev0** [comment]
> It should remain as private channel (I think only CNCF has permission to change the visibility). They can submit a PR or ask us in wf contributors channel.

Well, they will be submitting a PR anyway to add their name to the document. So, as a matter of process we could add them when we merge that? (assuming we don't forget)

**Comment by terrytangyuan** [comment]
Well currently it points to wf-contributors channel which is not correct (they don't necessarily need to join this channel). 

This could serve as a good reminder for us as well.

**Comment by juliev0** [comment]
> Well currently it points to wf-contributors channel which is not correct (they don't necessarily need to join this channel).
> 
> This could serve as a good reminder for us as well.

Yeah, it's a good question as to whether they should still join the `argo-wf-contributors` channel. Maybe it's good for them to be part of that anyway?

File: community/sustainability_effort.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ There are different expectations depending on the role:
 
 Note that the hours per week listed above is an average over time; it's fine to have weeks of no activity so long as the average meets expectations.
 
-Participants should join the [`#argo-wf-contributors`](https://cloud-native.slack.com/archives/C0510EUH90V) Slack channel.
+Participants should join the [`#argo-wf-contributors`](https://cloud-native.slack.com/archives/C0510EUH90V) Slack channel and ask to be invited to the private `#argo-wf-sustainability` channel.
```
Comment (juliev0) [inline community/sustainability_effort.md:29]:
```suggestion
Participants should join the [`#argo-wf-contributors`](https://cloud-native.slack.com/archives/C0510EUH90V) Slack channel. They should also be invited to the private `#argo-wf-sustainability` channel.
```

Comment (juliev0) [reply community/sustainability_effort.md:29]: what do you think of something like that?

Comment (juliev0) [reply community/sustainability_effort.md:29]: I feel like if I saw this and somebody forgot to invite me, I would ask ?

**Review by juliev0 — APPROVED** [review]

**Pull request merged**